README.md

```diff
@@ -1,72 +1,47 @@
 # archive_project
-Backup the pipeline results to S3
+Tool to back up the data produced by the Pathogen Informatics sequencing pipelines to an S3 server
```

Comment:
Maybe mention that it isn't all of the data, just that that can't be easily reproduced

Reply:
I mention this in the "introduction". I believe that this is sufficient as a one-liner.